addons/mail/static/src/core/common/mail_popout_service.js

```diff
@@ -58,13 +57,11 @@ export const mailPopoutService = {
                 pollClosedWindow();
                 externalWindow.document.write(window.document.head.outerHTML);
             }
-            if (element !== currentReplacement) {
+            if (app) {
                 reset();
-                if (element) {
-                    currentReplacement = element;
-                    element.after(anchor);
-                    externalWindow.document.body.append(element);
-                }
+                toggleVisibility();
+                toggleVisibilityFn = toggleVisibility;
+                app.mount(externalWindow.document.body);
             }
             if (triggerResize) {
                 ui.bus.trigger("resize");
```

Comment:
Not sure this code is needed anymore (if a t-if with reactive is used on the component)